easybuild/easyconfigs/a/Arrow/Arrow-11.0.0-gfbf-2022b.eb

```diff
@@ -53,7 +53,8 @@ local_install_pyarrow_cmds += "cd %(builddir)s/*arrow-%(version)s/python && expo
 local_install_pyarrow_cmds += "sed -i 's/numpy==[0-9.]*/numpy/g' pyproject.toml && "
 local_install_pyarrow_cmds += "Python3_ROOT_DIR=$EBROOTPYTHON "
 local_install_pyarrow_cmds += "PYARROW_CMAKE_OPTIONS='-DZSTD_LIB=$EBROOTZSTD/lib/libzstd.%s ' " % SHLIB_EXT
-local_install_pyarrow_cmds += "PYARROW_WITH_DATASET=1 PYARROW_WITH_PARQUET=1 pip install --prefix %(installdir)s ."
+local_install_pyarrow_cmds += "PYARROW_WITH_DATASET=1 PYARROW_WITH_PARQUET=1 "
+local_install_pyarrow_cmds += "pip install --prefix %(installdir)s --no-build-isolation ."
```

Comment:
should we also make this `python -m pip install ...` instead of just `pip install`?

Reply:
@branfosj thoughts on the suggestion from @jfgrimm?

Reply:
2f61086